src/olympia/addons/admin.py

```diff
@@ -120,10 +121,23 @@ def guid_slug(self, obj):
             slug = ugettext(u'- Add-on not on AMO -')
         return slug
 
+    def has_change_permission(self, request, obj=None):
+        # If an obj is passed, then we're looking at the individual change page
+        # for a replacement addon, otherwise we're looking at the list. When
+        # looking at the list, we also allow users with Addons:Edit - they
+        # won't be able to make any changes but they can see the list.
+        if obj is not None:
+            return super(ReplacementAddonAdmin, self).has_change_permission(
+                request, obj=obj)
+        else:
+            return (
+                acl.action_allowed(request, amo.permissions.ADDONS_EDIT) or
```

Comment:
I wonder if we should change this test to check for `ADMIN_ANY`, rather than `ADDONS_EDIT`.  Staff would still match (because they have `Admin:Tools`) but it would, admittedly, give view access also to any future holder of an `Admin:xxx` permission too.  Maybe that's okay as a general thing for "pseduo-admin" users?  Not a request to make a change, just noodling.

Reply:
I am not a fan. I really don't want to check for `ADMIN_ANY` anywhere, it's too broad. I agree `ADDONS_EDIT` is not ideal, but I think if it is to be changed, it should be for something more specific, not less.